packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -833,7 +833,6 @@ export class CodeModelBuilder {
     const operationName = sdkMethod.name;
     const httpOperation = sdkMethod.operation;
     const operationId = groupName ? `${groupName}_${operationName}` : `${operationName}`;
-    const operationGroup = this.codeModel.getOperationGroup(groupName);
```

Comment:
Hi @XiaofeiCao, I'm trying to learn from this pull request. 

The pr description states "remove operation groups from codemodel root" -  IIUC, calling `this.codeModel.getOperationGroup(...)` may add an `OperationGroup` object to the internal `codeModel.operationGroups` collection. By removing this function call, no operation-groups will ever get added to the `CodeModel` object. So "remove operation groups from codemodel root" effectively means not adding any operation-groups to `CodeModel`, Is my understanding correct?

 if so, is there any reason to keep the method `getOperationGroup(...)` in `CodeModel` class?

Reply:
Yes, your nailed it.

> if so, is there any reason to keep the method getOperationGroup(...) in CodeModel class?

I'm afraid we have to keep it for Swagger, where we only have `CodeModel.operationGroups`.

Reply:
It make sense now, thank you.